### DISCUSSION TITLE
Welcome to Wren AI Discussions!

### DISCUSSION THREAD

**Discussion by wwwy3y3** [Canner/WrenAI#326] [Announcements]
## 👋 Welcome!
  We're using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Senthilsivam41** [comment]
Hello, I found out WrenAI GenBI is very interesting. Assume that we got PostgresDB and it contains the relationship details in terms of Property Graph then whether this GenBI tool can be used out of box or do we need to do any customisation.

**Comment by nihar30-dev** [comment]
Hello,

There is an issue in this file:
https://docs.getwren.ai/oss/llms.md

While resolving an MR, merge conflict markers were accidentally included:

```text
<<<<<<< HEAD
genbi/archived/quickstart_claude.md
=======
.
.
engine/archived/quickstart_claude.md
>>>>>>> 454238f5 (update llm)
```

Please update/fix this as soon as possible.

**Reply by PaulChen79** [reply]
Hi @nihar30-dev  — thanks for catching this and for the clear repro 🙏

It's fixed and merged.

Thanks again for raising the issue.
